src/modules/Nfs.rb

```diff
@@ -557,22 +558,17 @@ def ProbeServers
       delim = ""
 
       # fallback from glibc (uses different field separators, grr :( )
+      # TODO: looks like glibc no longer contain this fallback?
       if !FileUtils.Exists(prog_name)
         prog_name = "/usr/sbin/rpcinfo"
         delim = "-d ' ' "
       end
 
       # #71064
       # this works also if ICMP broadcasts are ignored
-      cmd = Ops.add(
-        Ops.add(Ops.add(prog_name, " -b mountd 1 | cut "), delim),
-        "-f 2 | sort -u"
-      )
+      cmd = "#{prog_name} -b mountd 1 | /usr/bin/cut #{delim} -f 2 | /usr/bin/sort -u"
```

Comment:
NP: I usually do not like pipes with `echo | cat  | cut | grep | sort | uniq | ....`, I'd replace that with native Ruby if possible. (Unless the output is really huge, then `grep` would make sense of course.)

Reply:
well, here I ahve a bit problem as I do not have rpcinfo from glibc, so I cannot test it for all delims. So I prefer to be on safe side. Especially as this part is not tested.

Reply:
OK, makes sense.